lib/gui/lvgl/lv_conf.h

```diff
@@ -871,4 +872,8 @@ typedef void *lv_obj_user_data_t;
 
 #define LV_USE_WIN IS_ENABLED(CONFIG_LVGL_USE_WIN)
 
+#ifdef CONFIG_LVGL_USER_CONFIG_FILE
+#include CONFIG_LVGL_USER_CONFIG_FILE_INCLUDE
```

Comment:
Where is the `lv_conf_user.h` supposed to be located?

Is the user application supposed to expose the directory containing one as a global include dir in `CMakeLists.txt`?

Reply:
I have it in the root folder of my project, or in any folder, the path to which is registered in CMakeLists.txt via zephyr_include_directories.